canvas_modules/harness/package.json

```diff
@@ -73,7 +74,7 @@
     "react-inlinesvg": "4.1.3",
     "react-intl": "6.6.3",
     "react-redux": "7.2.8",
-    "react-refresh": "0.14.0",
+    "react-refresh": "^0.14.0",
```

Comment:
Any reason you're adding ^ to the versions?  I think most if not all these should be static versions.

Reply:
Hi @matthoward366 It got added when I ran `npm install -D @pmmmwh/react-refresh-webpack-plugin react-refresh` now I have reverted it.